assets/data-port/import/data/selectors.js

```diff
@@ -88,3 +89,36 @@ export const isReadyToStart = ( state ) => {
 
 	return hasUploaded && ! inProgress;
 };
+
+/**
+ * Get success results.
+ *
+ * @param {Object} state Current state.
+ *
+ * @return {Array} Success results.
+ */
+export const getSuccessResults = ( { done } ) =>
+	DONE_KEYS.map( ( key ) => ( {
+		key,
+		count: get( done, [ 'results', key, 'success' ], 0 ),
```

Comment:
I think @donnapep would eventually like this to be the sum of the `success` and `warning` counts.

Reply:
Good catch! Updated here: https://github.com/Automattic/sensei/pull/3379/commits/869d8a553985b2e84b73916bec46609b170b428b